.github/workflows/workflow-validate-repo-markdown.yml

```
@@ -31,42 +26,33 @@ jobs:
         submodules: recursive
     # Setup runtime environment
     # -------------------------
-    - name: Set up NodeJS
-      uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8
+    - name: Identify Markdown files
+      id: markdownfiles
+      run: echo "files=$(git ls-files "*.md" -z | grep  -v "${{ inputs.ignorePattern }}" --null-data | tr '\0' ' ')" >> $GITHUB_OUTPUT
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:110: Double quote to prevent globbing and word splitting

(shellcheck)